app/components/DocumentTitle/index.js

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+const DocumentTitle = ({ title }) => {
+  useEffect(() => {
+    document.title = title;
+  }, [title]);
```

Comment:
You could add a return function to this effect, which sets the document title back to a default when the component is removed. 

This is useful when a page is rendered which doesn't contain a DocumentTitle. In that case it won't show the wrong page title, but just a default.

Reply:
Done (https://github.com/Amsterdam/registraties/pull/107/commits/45fef4073bbc992f4437dd6d106db9914466f83f)